### PR TITLE
Create maintainer_guide.spa.md

### PR DESCRIPTION
**Pull request by Harshayu**
added Spanish translation

### CODE REVIEW

**Comment by CandidDeer** [comment]
@Syknapse Please let me know if we can merge this. Thx.

**Comment by Harshayu** [comment]
@Syknapse is there any changes needed ?
 if everything is ok then pls merge it

File: translations/maintainer_guide/maintainer_guide.spa.md [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+##Traducciones
+
+ "Este tutorial también está disponible en otros idiomas."
+
+ | [English](maintainer_guide.md)   | [Bangla](translations/maintainer_guide/maintainer_guide.ben.md) | [Chinese (Traditional)](/translations/maintainer_guide/maintainer_guide.zho-tc.md) | [German](translations/maintainer_guide/maintainer_guide.ger.md) | [Hindi](translations/maintainer_guide/maintainer_guide.hin.md) |
+| :---: | :---: | :---: | :---: | :---: |
+| [Italian](translations/maintainer_guide/maintainer_guide.ita.md) | [Japanese](translations/maintainer_guide/maintainer_guide.jpn.md) | [Portuguese](translations/maintainer_guide/maintainer_guide.por.md) | [Ukrainian](/translations/maintainer_guide/maintainer_guide.ukr.md) |
+[Spanish](/translation/maintainer_guide/maintainer_guide.spa.md)
+
+  Las traducciones para la documentación de proyectos son bienvenidas. Lee la [`Guía de Traducción`]
```
Comment (TenTraicion) [inline translations/maintainer_guide/maintainer_guide.spa.md:16]: the link isn't working, kindly remove the space/line break to make that work.

File: translations/maintainer_guide/maintainer_guide.spa.md [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+  (translations/README.md)para contribuir."
```
Comment (TenTraicion) [inline translations/maintainer_guide/maintainer_guide.spa.md:17]:
```suggestion
  Las traducciones para la documentación de proyectos son bienvenidas. Lee la [`Guía de Traducción`](translations/README.md) para contribuir."
```

**Review by TenTraicion — CHANGES_REQUESTED** [review]
@Harshayu, Thanks for your contribution. kindly make the required change and we are good to go.

**Pull request closed without merging**